jenkins/workers.py

```diff
@@ -0,0 +1,94 @@
+"""
+Use the jenkins queue and computer APIs to log info about workers.
+
+Usage:
+    `python jenkins/workers.py -j https://example.jenkins.com`
+
+Example Output:
+    [INFO] Counts of workers and queue length from Jenkins API:
```

Comment:
In the Jenkins console log, there is a timestamp on each line already.  I think adding one here might be noisy.